src/config/dynamic-theme-fixes.config

```diff
@@ -11792,6 +11792,55 @@ INVERT
 
 ================================
 
+manifold.markets
+
+CSS
+.bg-primary {
+    --darkreader-bg--tw-bg-opacity: 1 !important;
+    background-color: hsla(var(--p)/var(--darkreader-bg--tw-bg-opacity)) !important;
+}
+.bg-green-600 {
+    --darkreader-bg--tw-bg-opacity: 1 !important;
+    background-color: rgb(22 163 74/var(--darkreader-bg--tw-bg-opacity)) !important;
+}
+.bg-opacity-10 {
+    --darkreader-bg--tw-bg-opacity: 0.4 !important;
+}
+.border-primary {
+    --darkreader-border--tw-border-opacity: 1 !important;
+    border-color: hsla(var(--p)/var(--darkreader-border--tw-border-opacity)) !important;
+}
+.btn-outline:hover {
+    --darkreader-bg--tw-bg-opacity: 1 !important;
+    background-color: rgb(255 255 255/var(--darkreader-bg--tw-bg-opacity,1)) !important;
+    --darkreader-border--tw-border-opacity: 1 !important;
+    border-color: rgb(255 255 255/var(--darkreader-border--tw-border-opacity,1)) !important;
+    --darkreader-text--tw-text-opacity: 1 !important;
+    color: rgb(0 0 0/var(--darkreader-text--tw-text-opacity,1)) !important;
+}
+.hover\:bg-primary-focus:hover {
+    --darkreader-bg--tw-bg-opacity: 1 !important;
+    background-color: hsla(var(--pf)/var(--darkreader-bg--tw-bg-opacity)) !important;
+}
+.hover\:text-primary:hover {
+    --darkreader-text--tw-text-opacity: 1 !important;
+    color: hsla(var(--p)/var(--darkreader-text--tw-text-opacity)) !important;
+}
+.hover\:text-white:hover {
+    --darkreader-text--tw-text-opacity: 1 !important;
+    color: rgb(255 255 255/var(--darkreader-text--tw-text-opacity)) !important;
+}
+.text-primary {
+    --darkreader-text--tw-text-opacity: 1 !important;
+    color: hsla(var(--p)/var(--darkreader-text--tw-text-opacity)) !important;
+}
+.text-white {
+    --darkreader-text--tw-text-opacity: 1 !important;
+    color: rgb(255 255 255/var(--darkreader-text--tw-text-opacity)) !important;
+}
```

Comment:
Please remove the variables and instead just use the values in the code instead.

Reply:
Fixed! See my latest commit.

Reply:
@Gusted Let me know if you want anything else changed or if it looks good to you.